lms/djangoapps/discussion_api/views.py

```diff
@@ -157,10 +157,19 @@ class ThreadViewSet(DeveloperErrorViewMixin, ViewSet):
         * view: "unread" for threads the requesting user has not read, or
             "unanswered" for question threads with no marked answer. Only one
             can be selected.
+        * requested_fields: (list) Indicates which additional fields to return
+          for each thread. (supports 'profile_image')
 
         The topic_id, text_search, and following parameters are mutually
         exclusive (i.e. only one may be specified in a request)
 
+    **GET Thread Parameters**:
+
+        * thread_id: The id of the thread
+
+        * requested_fields: (list) Indicates which additional fields to return
+          for each thread. (supports 'profile_image')
```

Comment:
If I'm reading this correctly, thread_id is part of the main URL, while requested_fields is an optional parameter.  The distinction should be clear in the documentation.  Also, in the example URLs above, can you fix the formatting of the URLs so it's clear that thread_id is a placeholder:  `GET /api/discussion/v1/threads/{thread_id}`.